configure.ac

```diff
@@ -2069,6 +2069,17 @@ AS_CASE([$enable_debug_runtime],
   [no], [debug_runtime=false],
   [debug_runtime=true])
 
+## Determine if system stack overflows can be detected
+
+AC_MSG_CHECKING([whether stack overflows can be detected])
+
+AS_CASE([$arch,$system],
+  [i386,linux_elf|amd64,linux|amd64,macosx \
+    |amd64,openbsd|i386,bsd_elf|arm64,linux|arm64,macosx],
+    [AC_DEFINE([HAS_STACK_OVERFLOW_DETECTION])
+    AC_MSG_RESULT([yes])],
+  [AC_MSG_RESULT([no])])
+
```

Comment:
This shouldn't be merged until someone has run `autoconf` and committed the new `configure` script on a machine with recent autoconf installed.